examples/mining_ar.py

```diff
@@ -0,0 +1,128 @@
+          'in the dataset with minconf=1. If no minsup is specified, '
+          'the default value of minsup=0 is used.\n')
+    print_ars(algo.get_ars())
+    print("\n['Eggs'] -> ['Milk'] with confidence 1 means that whenever eggs "
+          'are found in the receipt, milk will '
+          f'{COLOR_CODES["green"]}always{COLOR_CODES["default"]} '
+          'be present as well. The same holds true for all other rules with '
+          f'{COLOR_CODES["bold_green"]}confidence 1{COLOR_CODES["default"]}.')
+
+    print("\n\nNow, let's examine the same dataset with "
+          f'{COLOR_CODES["yellow"]}minconf=0.7{COLOR_CODES["default"]}')
+    algo.execute(minconf=0.7)
+    print_ars(algo.get_ars())
+    print("\n['Milk'] -> ['Eggs'] with confidence 0.75 means that when milk "
+          'is found in the receipt, the chance of eggs being '
+          'present amounts to 75%. So, customers are '
+          f'{COLOR_CODES["bold_yellow"]}likely{COLOR_CODES["default"]} to buy eggs '
+          f'with milk, but {COLOR_CODES["bold_red"]}not always.{COLOR_CODES["default"]}')
+
+    print('\nLet us turn to the next issue. You can observe that there are too many '
+          'association rules found in this small dataset. This happens because we did not '
+          'set up the support value. Due to this fact, the system discovers '
+          'all association rules, even those which are present only '
+          'once in the dataset. Now let\'s see the results with '
+          f'{COLOR_CODES["yellow"]}minsup=0.5{COLOR_CODES["default"]} and '
+          f'{COLOR_CODES["yellow"]}minconf=0.5{COLOR_CODES["default"]}.\n')
+    algo.execute(minsup=0.5, minconf=0.5)
+    print_ars(algo.get_ars())
+    print('\nNow you can see that the number of association rules have decreased '
+          'significantly. This happened due to minsup being set to 0.5, which basically '
+          'means that association rule needs to are present in 50% of all transactions '
+          "in order to be included in the result. Unfortunately, you can't get support "
+          'for each individual association rule with Desbordante.\n'
+          '\nA typical approach to controlling the algorithm is to employ "usefulness", '
+          'which is defined as confidence * support. In the last example, we set up min '
+          '"usefulness" = 0.5 * 0.5 = 0.25. \n\nNow let\'s try with '
+          f'{COLOR_CODES["green"]}minsup=0.7{COLOR_CODES["default"]}, '
+          f'{COLOR_CODES["green"]}minconf=0.7{COLOR_CODES["default"]} '
+          f'and {COLOR_CODES["bold_yellow"]}"usefulness"=0.49{COLOR_CODES["default"]}\n')
+    algo.execute(minsup=0.7, minconf=0.7)
+    print_ars(algo.get_ars())
+    print('\nSo, now the total number of returned association rules is only four. '
+          'We reduced the amount of "noisy" information in our output. However, '
+          'this is just a single example of how you can evaluate the usefulness of data. '
```

Comment:
You typically do not "evaluate" a parameter, as you are setting it to get some other results. You could omit this sentence or rewrite it if you deem this information to be of use to the user